pkg/translator/prometheusremotewrite/testutils_test.go

```diff
@@ -184,11 +184,16 @@ func getTimeSeriesWithSamplesAndExemplars(labels []prompb.Label, samples []promp
 	}
 }
 
-func getHistogramDataPointWithExemplars(t *testing.T, time time.Time, value float64, traceID string, spanID string, attributeKey string, attributeValue string) pmetric.HistogramDataPoint {
+func getHistogramDataPointWithExemplars[V int64 | float64](t *testing.T, time time.Time, value V, traceID string, spanID string, attributeKey string, attributeValue string) pmetric.HistogramDataPoint {
 	h := pmetric.NewHistogramDataPoint()
 
 	e := h.Exemplars().AppendEmpty()
-	e.SetDoubleValue(value)
+	switch v := (interface{})(value).(type) {
```

Comment:
```suggestion
	switch v := (any)(value).(type) {
```
Not sure why the formatter didn't catch this, but I'm pretty sure we're standardizing on `any` instead of `interface{}` throughout the repository. Would this work here?

Reply:
@evan-bradley this decision is just related to a style thing or is something else?

Reply:
It's stylistic, yeah. I believe `any` and `interface{}` are exactly equivalent, `any` is just a shorter and clearer name.

Reply:
Do you think that would be nice to investigate why the lint didn't pop up any constraint about it?